TraitsExpanded/TraitSystemBehavior.cs

```diff
@@ -39,36 +44,52 @@ public TraitSystemBehavior()
         private void Init()
         {
             OnCampaignTick += ValidateHeroList;
+            CampaignEvents.OnGameLoadedEvent.AddNonSerializedListener(this, starter =>
+              {
+                  GivePlayerTraitSet(new BigCharacterTraitSet(null));
+               });
+        }
+
+        private void OnGameStart(CampaignGameStarter starter)
+        {
+            
+        }
+
+        private void PopulateTraitSets()
+        {
+            AllTraitSets.Add(new BigCharacterTraitSet(null));
         }
 
         public void Tick()
         {
             if (MissionTickTimer.Check(Time.ApplicationTime))
             {
-                OnMissionTick.Invoke(new MissionStatusInfo());
+                OnMissionTick?.Invoke(new MissionStatusInfo());
                 MissionTickTimer.Reset(Time.ApplicationTime, MissionTickTime);
             }
+
             if (CampaignTickTimer.Check(Time.ApplicationTime))
             {
-                OnCampaignTick.Invoke(new CampaignStatusInfo());
+                OnCampaignTick?.Invoke(new CampaignStatusInfo());
                 CampaignTickTimer.Reset(Time.ApplicationTime, CampaignTickTime);
             }
+
             if (GameTickTimer.Check(Time.ApplicationTime))
             {
-                OnGameTick.Invoke(Game.Current);
+                OnGameTick?.Invoke(new GameStatusInfo());
                 GameTickTimer.Reset(Time.ApplicationTime, MissionTickTime);
             }
         }
 
-        public void CheckRequirements()
-        {
-            
-        }
 
         private void ValidateHeroList(CampaignStatusInfo statusInfo)
```

Comment:
this method seems to do more than validation, consider amending the name.

also we can move the notes about these APIs from the miro board to documentation here, to make things more self-explanatory in the code.

Reply:
For sure, I'll move them over the weekend